src/applicationContext.ts

```diff
@@ -25,6 +25,11 @@ import {LearningProvider} from './learning-catalogue/model/learningProvider'
 import {CancellationPolicy} from './learning-catalogue/model/cancellationPolicy'
 import {TermsAndConditions} from './learning-catalogue/model/termsAndConditions'
 import {Course} from './learning-catalogue/model/course'
```

Comment:
ApplicationContext getting bigger and bigger, can we break this down to Components, please?

Reply:
Fixing in later PR